cpu/esp8266/Makefile.include

```diff
@@ -0,0 +1,175 @@
+# check some environment variables first
+ifndef NEWLIB_DIR
+    $(info NEWLIB_DIR should be defined as /path/to/newlib directory)
+    $(info NEWLIB_DIR is set by default to /opt/esp/newlib-xtensa)
+    export NEWLIB_DIR=/opt/esp/newlib-xtensa
+endif
+
+ifndef SDK_DIR
+    $(info SDK_DIR should be defined as /path/to/sdk directory)
+    $(info SDK_DIR is set by default to /opt/esp/esp-open-sdk/sdk)
+    export SDK_DIR=/opt/esp/esp-open-sdk/sdk
+endif
+
+export CPU ?= esp8266
+export TARGET_ARCH ?= xtensa-lx106-elf
+
+INCLUDES += -I$(NEWLIB_DIR)/$(TARGET_ARCH)/include
+INCLUDES += -I$(RIOTBOARD)/common/$(CPU)/include
+INCLUDES += -I$(RIOTCPU)/$(CPU)
+INCLUDES += -I$(RIOTCPU)/$(CPU)/espressif
+
+CFLAGS  += -DRIOT_OS -DESP_OPEN_SDK
```

Comment:
can this be changed to re-use ```RIOT_VERSION```, which is always available?

Reply:
Sure. I was simply looking for a possibility to check whether we are in compilation of RIOT.